packages/shared/src/components/MainLayout.tsx

```diff
@@ -111,47 +99,28 @@ export default function MainLayout({
         {!showOnlyLogo && !loadingUser && (
           <>
             {user ? (
-              <>
-                <LinkWithTooltip
-                  href={`${process.env.NEXT_PUBLIC_WEBAPP_URL}bookmarks`}
-                  passHref
-                  prefetch={false}
-                  tooltip={{ placement: 'bottom', content: 'Bookmarks' }}
-                >
-                  <HeaderButton
-                    tag="a"
-                    icon={<BookmarkIcon />}
-                    className="btn-tertiary"
-                  />
-                </LinkWithTooltip>
-                {afterBookmarkButtons}
-                <ProfileButton onShowDndClick={onShowDndClick} />
-              </>
+              <ProfileButton onShowDndClick={onShowDndClick} />
             ) : (
-              <>
-                <SimpleTooltip placement="bottom" content="Bookmarks">
-                  <HeaderButton
-                    icon={<BookmarkIcon />}
-                    onClick={() => showLogin('bookmark')}
-                    className="btn-tertiary"
-                  />
-                </SimpleTooltip>
-                {afterBookmarkButtons}
-                <Button
-                  onClick={() => showLogin('main button')}
-                  className="btn-primary"
-                >
-                  Login
-                </Button>
-              </>
+              <Button
+                onClick={() => showLogin('main button')}
+                className="btn-primary"
+              >
+                Login
+              </Button>
             )}
           </>
         )}
-        {showRank && windowLoaded && (
-          <HeaderRankProgress className="absolute right-0 -bottom-px left-0 z-rank my-0 mx-auto transform translate-y-1/2" />
-        )}
       </header>
-      {children}
+      <main className="flex flex-row">
+        <aside
+          className="bg-theme-float transition-all transform duration-500 ease-in-out"
+          onClick={() => toggleOpenSidebar()}
+          style={{ width: openSidebar ? 240 : 44 }}
```

Comment:
I think there are classes suitable for this width - namely `w-60` and `w-11`. Just to keep things coming from one place.